providers/google/src/airflow/providers/google/cloud/hooks/spanner.py

```diff
@@ -427,3 +430,40 @@ def _execute_sql_in_transaction(transaction: Transaction, queries: list[str]) ->
             rc = transaction.execute_update(sql)
             counts[sql] = rc
         return counts
+
+    def _get_openlineage_authority_part(self, connection):
```

Comment:
adding type definitions here would be useful

```suggestion
    def _get_openlineage_authority_part(self, connection: Connection) -> str | None:
```

Reply:
fixed